spec/util/miq-password_spec.rb

```diff
@@ -353,6 +354,54 @@
     end
   end
 
+  describe ".encrypt_hash" do
+    it "should not encrypt blank" do
+      options = { "my_password" => "" }
+      new = MiqPassword.encrypt_hash(options)
+      expect(new["my_password"]).to be_empty
+      expect(new.keys.include?("password::my_password")).to be false
+    end
+
+    it "should add 'password::' prefix" do
+      options = { "my_password" => "secret" }
+      new = MiqPassword.encrypt_hash(options)
+      expect(new.keys.first).to eq("password::my_password")
+      expect(new["password::my_password"]).to be_encrypted("secret")
+      expect(new.keys.include?("my_password")).to be false
```

Comment:
Please don't use keywords as variable names, it makes things like this ambiguous.